tests/ut_mdesktopentry.cpp

```diff
@@ -4,6 +4,10 @@
 #include <QtCore/QTextStream>
 #include <QtCore/QTranslator>
 
+#if (QT_VERSION > QT_VERSION_CHECK(6, 0, 0))
+#include <QRegularExpression>
```

Comment:
QRegularExpression but then the code uses QRegExp.

Just switch to this class and use it on both qt versions. Shouldn't need ifdeffing.